src/main/java/mate/jdbc/dao/DriverDaoImpl.java

```diff
@@ -37,9 +37,9 @@ public Driver create(Driver driver) {
 
     @Override
     public Optional<Driver> get(Long id) {
-        String query = "SELECT id, name, license_number "
+        String query = "SELECT id, name, licenseNumber "
```

Comment:
naming was valid

Reply:
[SQL](https://www.sqlstyle.guide/) code style demands in case where space needed between words - use snake_case